assets/styl/_modules/_buttons.styl

```diff
@@ -52,32 +94,34 @@ $btn-twitter-color ?= #00a7e7
 $btn-whatsapp-color ?= #01a85a
 
 .btn-icon
-    /* generic styles */
     display inline-block
     width 1.6667em
     height 1.6667em
     border-radius 60em // oversize radius to keep it round
-    background-color $color-secondary
     font-size 1em
     line-height 1.833333em
     text-align center
-    color white
```

Comment:
Because setting a default `background-color` on `.btn-{color}` class doesn't override the default `.btn` background-color.
Removing them 1) doesn't break the default display 2) avoids having something like `.btn.btn-{color}` or `!important` in the css.

Is it ok ?